conf/spgwu.json

```diff
@@ -1,7 +1,8 @@
 {
-    "": "Vdev support. Enable `\"mode\": \"af_xdp\"` to enable AF_XDP mode, or `\"mode\": \"af_packet\"` to enable AF_PACKET mode",
+    "": "Vdev or sim support. Enable `\"mode\": \"af_xdp\"` to enable AF_XDP mode, or `\"mode\": \"af_packet\"` to enable AF_PACKET mode, or `\"mode\": \"sim\"` to generate synthetic traffic from BESS's Source module",
```

Comment:
what are the equivalent changes on docker_setup.sh side?

Reply:
then you have to populate the route table and connect mac update to measure/sink too. Right?

Reply:
> then you have to populate the route table and connect mac update to measure/sink too. Right?

There is no route module in `sim` mode. The packets go through this pipeline:
```
sgi_source -> sgi_rewrite -> sgi_timestamp -> sgiFastBPF -> sgiRxIPCksum ->
sgiRxUDPCksum ->  DLMACDstMatch -> sgiIP4Defrag -> ueLookup -> PktParse ->
PDRLookup -> PreQoSCounter -> FARLookup -> ExecuteFAR -> EtherEncapTrim ->
GTPUEncap -> S1UEtherAdd -> PostQoSCounter -> OuterUDPCsum -> OuterIPCsum ->
s1uMerge -> s1u_measure -> s1uIP4Frag -> s1u_out (Sink())
```